code/modules/uplink/uplink_items/implant.dm

```diff
@@ -9,8 +9,8 @@
 
 /datum/uplink_item/implants/freedom
 	name = "Freedom Implant"
-	desc = "An implant injected into the body and later activated at the user's will. It will attempt to free the \
-			user from common restraints such as handcuffs."
+	desc = "Can be activated to release common restraints such as handcuffs, legcuffs, and even bolas tethered around the legs. \
+			Implant has enough energy for 4 uses before it becomes inert and harmlessly self-destructs."
```

Comment:
```suggestion
	desc = "Can be activated to release common restraints such as handcuffs, legcuffs, and even bolas tethered around the legs. \
		Implant has enough energy for [initial(uses)] uses before it becomes inert and harmlessly self-destructs."
```

Reply:
This is the uplink datum, I don't think I can access the uses value unless I have an instance of a freedom implant to get it from.

Should I make the freedom implant uses value a define instead?

Reply:
Yeah a shared define sounds sensible in that case